packages/composables/src/useWishlist.test.ts

```diff
@@ -1,37 +1,159 @@
 import { describe, expect, it, vi } from "vitest";
 import { useWishlist } from "./useWishlist";
+import { useSyncWishlist } from "./useSyncWishlist";
+import { useUser } from "./useUser";
 import { useSetup } from "./_test";
+import { computed, ref } from "vue";
+import type { operations } from "#shopware";
+
+vi.mock("./useLocalWishlist.ts", () => ({
+  useLocalWishlist() {
+    return {
+      getWishlistProducts: () => undefined,
+      items: {
+        value: [
+          "local-testId-1",
+          "local-testId-2",
+          "local-testId-3",
+          "local-testId-4",
+          "local-testId-5",
+          "local-testId-6",
+          "local-testId-7",
+          "local-testId-8",
+          "local-testId-9",
+          "local-testId-10",
+          "local-testId-11",
+          "local-testId-12",
+          "local-testId-13",
+          "local-testId-14",
+          "local-testId-15",
+          "local-testId-16",
+          "local-testId-17",
+        ],
+      },
+      clearWishlist: () => undefined,
+      totalWishlistItemsCount: ref(17),
+    };
+  },
+}));
+
+vi.mock("./useSyncWishlist.ts");
+vi.mock("./useUser");
 
 describe("useWishlist - not logged in user", () => {
-  vi.mock("./useLocalWishlist.ts", () => ({
-    useLocalWishlist() {
-      return {
-        getWishlistProducts: () => undefined,
-        items: { value: ["local-testId333"] },
-        clearWishlist: () => undefined,
-      };
-    },
-  }));
-
-  it("mergeWishlistProducts", async () => {
+  it("mergeWishlistProducts", () => {
+    vi.mocked(useUser).mockReturnValue({
+      isLoggedIn: computed(() => false),
+      isGuestSession: computed(() => true),
+      // @ts-ignore
+      canSyncWishlist: { value: false },
```

Comment:
ts-ignore is not a solution, useUser doesn't have `canSyncWishlist` so shouldn't be in the mock
```suggestion
```